mediacloud/mediawords/util/html.py

```diff
@@ -34,12 +39,13 @@ def link_canonical_url_from_html(html: str, base_url: str = None) -> Optional[st
     return None
 
 
-def meta_refresh_url_from_html(html: str, base_url: str = None) -> Optional[str]:
+def meta_refresh_url_from_html(html: str, base_url: Optional[str] = None) -> Optional[str]:
     """From the provided HTML, determine the <meta http-equiv="refresh" /> URL (if any)."""
+    def __get_meta_refresh_url_from_tag(inner_tag: str, inner_base_url: Optional[str]=None) -> Optional[str]:
+        """Given a <meta ...> tag, return the url from the content="url=XXX" attribute.
 
-    def __get_meta_refresh_url_from_tag(inner_tag: str, inner_base_url=None) -> Optional[str]:
-        """Given a <meta ...> tag, return the url from the content="url=XXX" attribute.  return undef if no such url is
-        found."""
+        return undef if no such url isfound.
```

Comment:
return `None` if no such url is found`

Reply:
Surprised this is the only of those that snuck through!